src/klh10.c

```diff
@@ -2760,6 +2761,7 @@ fc_lights(struct cmd_s *cm)
     printf("?Bad address\n");
 #endif /* KLH10_DEV_LITES */
 }
+#endif
 
```

Comment:
YES to these 2 changes, But remove the inner ifdefs, and move the comment on the `#endif` to the remaining endif.